pyhap/accessory_driver.py

```diff
@@ -157,6 +181,68 @@ def __init__(self, *, address=None, port=51234,
         network_tuple = (self.state.address, self.state.port)
         self.http_server = HAPServer(network_tuple, self)
 
+    def start(self):
+        """Start the event loop and call `start_pyhap`.
+
+        Pyhap will be stopped gracefully on a KeyBoardInterrupt.
+        """
+        try:
+            logger.info('Starting the event loop')
+            self.add_job(self.start_pyhap)
+            self.loop.run_forever()
+        except KeyboardInterrupt:
```

Comment:
> To be honest I haven't tested pyhap on windows. Ever. Have you?

No :smile: I only know that `KeyboardInterrupt` works for Windows as well.

> Signal handlers

`driver.stop` does still work, not just the `KeyboardInterrupt`. I don't have that much experience with running programs as `demon processes`, but I think HA doesn't use any signal handler and it still exits gracefully. Can you try stopping the `service` without any signal handlers?

> `driver.loop.add_signal_handler`

I would prefer the current solution is basically the same if not easier, since I don't need an extra method.

Reply:
If the service exits gracefully even without the signal handler, I think we can remove `driver.signal_handler` as well.